tests/ctst/steps/replication.ts

```diff
@@ -89,13 +89,16 @@ When('the job to replicate existing objects with status {string} is executed',
         await createAndRunPod(this, podManifest);
     });
 
-Then('the object should eventually {string} replicated', { timeout: 360_000 },
-    async function (this: Zenko, replicate: 'be' | 'fail to be') {
+Then('the object replication should {string} within {int} seconds', { timeout: 600_000 },
```

Comment:
do we actually need to increase timeout from 6 to 10 minutes?
did you experience cases of timeout?